src/CopyrighFixer/CopyrighFixer/sign.cpp

```diff
@@ -37,4 +45,115 @@ const QString &Signature::getMessage() const {
     return _customMessage;
 }
 
+bool Signature::fromJson(const QString &pathToFile) {
+
+    QFileInfo signFile(pathToFile);
+    if (signFile.exists() && signFile.isFile()) {
```

Comment:
You create two object on stack it is QFileInfo and QFile. please use only the QFile object.
```suggestion
    QFile file(pathToFile);
    if (file.exists()) {
```